frontend/packages/dev-console/src/components/edit-application/__tests__/edit-application-data.ts

```diff
@@ -663,4 +680,478 @@ export const knExternalImageValues: DeployImageFormData = {
   },
   searchTerm: 'openshift/hello-openshift',
   serverless: { scaling: { concurrencylimit: '', concurrencytarget: '', maxpods: '', minpods: 0 } },
+  healthChecks: {
+    readinessProbe: healthChecksDefaultValues,
+    livenessProbe: healthChecksDefaultValues,
+    startupProbe: healthChecksDefaultValues,
+  },
+};
+
+export const appResourcesWithHealthChecksEnabled: AppResources = {
```

Comment:
Why are you creating mock resources again? You can simply use the already created resources and add the probes data in them wherever required. This is creating so much duplicate code and the file size has become huge.